glite/version/config.pan

```diff
@@ -15,8 +15,17 @@ include { NODE_GLITE_VERSION_DB };
 
 # Find version to run on this node based on node name or OS version
 # and update loadpath accordingly.
+# If the version is defined to the empty string, it means that gLite must not be configured
+variable TEST = if ( is_defined(NODE_GLITE_VERSION[escape(FULL_HOSTNAME)]) ) {
```

Comment:
This TEST variable is redundant. It looks like it's purely to facilitate calling debug()? In which case, why not move those statements inside the block that defines NODE_GLITE_VERSION_LOADPATH? 

Also please use format() to make the debug statements easier to read.